internal/index/manager/manager.go

```diff
@@ -821,6 +829,27 @@ func (mgr *Manager) getIndexesCopy(start int) ([]*index.Reader, indexReleaser) {
 	return indexes, mgr.lock(indexes)
 }
 
+func (mgr *Manager) SetClientConfig(config ClientConfig) ClientConfig {
+	mgr.clientConfig = config
+	return mgr.ClientConfig()
+}
+
+func (mgr *Manager) ClientConfig() ClientConfig {
+	c := make(chan ClientConfig)
+	mgr.jobs <- func() {
+		locks := uint(0)
```

Comment:
You don't need to count those since you don't output that number. Looks like a copy&paste leftover from `Statistics()`.